charts/controlplane/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: controlplane
 description: Deploys all microservices associated with managing and controlling the roclub connectors
 type: application
-version: 2.1.4
-appVersion: "2.1.4"
+version: 2.3.0
```

Comment:
where is that version coming from?

Update: Nils prepared for the version we will have onve the livekit backend is moved here.